packages/slider/src/use-range-slider.ts

```diff
@@ -347,9 +347,14 @@ export function useRangeSlider(props: UseRangeSliderProps) {
   const onPanSessionStart = (event: AnyPointerEvent) => {
     const pointValue = getValueFromPointer(event) || 0
     const distances = value.map((val) => Math.abs(val - pointValue))
-    const isThumbStacked = new Set(distances).size !== distances.length
     const closest = Math.min(...distances)
     let index = distances.indexOf(closest)
+
+    // check if the clicked thumb is stacked by checking if there are multiple
+    // thumbs at the same distance
+    const isThumbStacked =
+      distances.filter((distance) => distance === closest).length > 1
```

Comment:
```suggestion
const isThumbStacked = distances.some((distance) => distance === closest)
```

Reply:
`.some` will return `true` even for 1 occurence. Need to check if multiple Thumbs are in the same location. In this case, `.some` will always return true

Reply:
Oh...we need more than one occurence. Cool then. Thanks for working on this